x-pack/plugins/index_management/public/application/sections/home/data_stream_list/data_stream_table/data_stream_table.tsx

```diff
@@ -44,14 +52,36 @@ export const DataStreamTable: React.FunctionComponent<Props> = ({
     }),
     truncateText: true,
     sortable: true,
-    render: (name: DataStream['name']) => {
+    render: (name: DataStream['name'], dataStream: DataStream) => {
       return (
-        <EuiLink
-          data-test-subj="nameLink"
-          {...reactRouterNavigate(history, getDataStreamDetailsLink(name))}
-        >
-          {name}
-        </EuiLink>
+        <Fragment>
+          <EuiLink
+            data-test-subj="nameLink"
+            {...reactRouterNavigate(history, getDataStreamDetailsLink(name))}
+          >
+            {name}
+          </EuiLink>
+          {isManagedByIngestManager(dataStream) ? (
+            <Fragment>
+              &nbsp;
+              <EuiToolTip
+                content={i18n.translate(
+                  'xpack.idxMgmt.dataStreamList.table.managedDataStreamTooltip',
+                  {
+                    defaultMessage: 'Created and managed by Ingest Manager',
```

Comment:
```suggestion
                    defaultMessage: 'Created and managed by Fleet',
```

Reply:
Same comment as before.

Reply:
Some suggestions that avoid `Fleet`:

* `Created and managed by Elastic Agent`
* `Created and managed by an agent`